tests/framework/helpers/ArrayHelperTest.php

```diff
@@ -51,6 +51,9 @@ protected function setUp()
 
     public function testToArray()
     {
+        $dataArrayable = $this->getMock('yii\\helpers\\ArrayHelper');
+        $this->assertEquals([], ArrayHelper::toArray($dataArrayable));
```

Comment:
This is test line https://github.com/yiisoft/yii2/blob/master/framework/helpers/BaseArrayHelper.php#L87 I made a mistake in the name of the class should be interface `yii\base\Arrayable`.